e2e-tests/cypress/tests/cypress/e2e/UserProfile.ChangePassword/the_user_is_presented_a_{string}_message.ts

```diff
@@ -0,0 +1,11 @@
+import { When } from "@badeball/cypress-cucumber-preprocessor";
+import { Toasts } from "../models/Toasts";
+
+When("the user is presented a {string} message", (type: string) => {
```

Comment:
Do we need such terrible file names?

Reply:
No, we do not.
The file naming and structuring of the step definitions has been solved in https://github.com/gradido/gradido/commit/a12bb9e09b8611b9e3211bf3c55c10670ab2a920.
Is that acceptable?